website/src/app/kb/architecture/core-components/readme.mdx

```diff
@@ -0,0 +1,197 @@
+import SupportOptions from "@/components/SupportOptions";
+import NextStep from "@/components/NextStep";
+import Image from "next/image";
+import Link from "next/link";
+import Alert from "@/components/DocsAlert";
+
+# Architecture: Core Components
+
+Firezone consists of a few core components that work together in unison to
+deliver secure, reliable access wherever your workforce is located.
+
+These components will be referred to throughout the documentation, so it's
+important to understand what each one does and how they interact with each
+other.
+
+## Overview
+
+Components in Firezone can be divided into two categories:
+
+1. **Firezone-managed**: Components that we host and operate for you.
+1. **User-managed**: Components you host and/or operate yourself.
+
+<Link
+  target="_blank"
+  href="/images/kb/architecture/core-components/component-diagram.svg"
+>
+  <Image
+    src="/images/kb/architecture/core-components/component-diagram.svg"
+    alt="Firezone core component diagram"
+    width={1200}
+    height={1200}
+    className="p-16"
+  />
+</Link>
+
+Each component is described in more detail below.
+
+## Firezone-managed components
+
+These are components we host and operate in our cloud environment.
+
+### Admin portal
+
+`https://app.firezone.dev`
+
+The admin portal is the web-based interface where you configure and manage
+Firezone for your organization.
+
+It provides a single pane of glass for configuring authentication, Resources,
+Gateways, Policies, and more.
+
+The admin portal is delivered as a managed SaaS application that's load-balanced
+globally for high availability.
+
+### Control plane API
+
+`wss://api.firezone.dev`
+
+The control plane API is a WebSocket-based, realtime API that Relays, Gateways,
+and Clients connect to for configuration and policy updates. It's responsible
+for distributing WireGuard keys to relevant peers and serves as the signaling
+layer for Firezone's STUN/TURN implementation.
+
+All configuration updates made in the admin portal are pushed in realtime to
+connected Clients and Gateways so they always have up-to-date information about
+which Resources are available and which Gateways to use for connecting to them.
+This informs our approach to high availability in Firezone: if a Gateway goes
+offline, Clients will immediately discontinue using it for new connections.
+
+WebSocket connections are authenticated using a cryptographic token with a
+lifetime that varies depending on the type of connection. See
+[security](/kb/architecture/security) for more information regarding token
```

Comment:
Is this page still to come?  When I clicked on it I was taken to a 'not found' page.

Reply:
Ah, should go to `/kb/authenticate`, fixed